### DISCUSSION TITLE
Android 14 you cant allow the other device to act like your self

### DISCUSSION THREAD

**Discussion by MrBLif** [rustdesk/rustdesk#9518] [General]
### Bug Description

You can't set the allowance to control a device with Android 14. 

### How to Reproduce

Install it on android 14 by the website
Run it
Go to "share screen"
Try  to toggle the allowance to Control

### Expected Behavior

That it's possible to Control the device from another one

### Operating system(s) on local side and remote side

Android 14 (both) 

### RustDesk Version(s) on local side and remote side

1.3.1 - > 1.3.1

### Screenshots

![IMG_20240929_100254744~2](https://github.com/user-attachments/assets/68a0af82-efcd-4aa9-9c5d-28070d65fd35)

**Comment by xlionjuan** [comment]
https://support.google.com/android/answer/12623953?hl=en
